specification/resources/serverless/responses/namespace_created.yml

```
@@ -0,0 +1,11 @@
+description: The response will be a JSON object with a key called `namespace`.
+  The value of this will be an object that contains the standard attributes
+  associated with a namespace.
+
+content:
```

You can use the shared ratelimit-limit components here:

```suggestion
headers:
  ratelimit-limit:
    $ref: '../../../shared/headers.yml#/ratelimit-limit'
  ratelimit-remaining:
    $ref: '../../../shared/headers.yml#/ratelimit-remaining'
  ratelimit-reset:
    $ref: '../../../shared/headers.yml#/ratelimit-reset'

content:
```